### DISCUSSION TITLE
normal mode in special modes, like dired

### DISCUSSION THREAD

**Discussion by dxlr8r** [meow-edit/meow#188] [Q&A] [answered]
I have read the tutorial, and the point about overwriting and using `hyper`. But it would also be nice if one could enter normal mode, and use the navigational bindings available there as normal.

Perhaps it is already possible?

**Comment by DogLooksGood** [comment, marked as answer]
There's a command called `meow-temp-normal` can be used to switch to NORMAL state in special mode, (switch back with `meow-insert/append`). but I don't think it is necessary.  For example, in dired, you only want arbitrary movement when you are going to edit with `wdired`. And when you switch to `wdired` , NORMAL will be used by default.

Additionally, vanilla Emacs keybindings are always available. Since they are not frequently used in special mode, it's okay to use keybinding with modifiers.
